implementations/micrometer-registry-appoptics/src/main/java/io/micrometer/appoptics/AggregateMeasurement.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.appoptics;
+
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.lang.Nullable;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Represents pre-aggregated measurements
+ * https://docs.appoptics.com/api/?shell#create-a-measurement
+ *
+ * @author Hunter Sherman
+ */
+public class AggregateMeasurement implements Measurement {
+
+    private final String name;
+    private final double sum;
+    private final long count;
+    @Nullable
+    private final Double max;
+    private final Map<String, String> tags;
+
+    private AggregateMeasurement(Builder builder) {
+        name = builder.name;
+        sum = builder.sum;
+        count = builder.count;
+        max = builder.max;
+        tags = builder.tags;
+    }
+
+    public String appendJson(StringBuilder json) {
+        json.append("{\"name\":\"").append(name)
+            .append("\",\"sum\":").append(sum)
+            .append(",\"count\":").append(count);
+        if(null != max) {
+            json.append(",\"max\":").append(max);
+        }
+        if(null != tags && !tags.isEmpty()) {
+            json.append(",\"tags\":").append(Measurement.tagsJson(tags));
+        }
+        return json.append("},").toString();
```

Comment:
@mheffner AFAIK, no and I was writing a *limited* version for it but there were some glitches with `DoubleFormat` and `StringEscapeUtils` to generalize it. I was blocked there but thinking to handle them with special options. I can create a PR for a *generalized* one first if it will help.